### PR TITLE
[BUGFIX] Add nav_hide to rootline

### LINKED ISSUE #1408: VHS Version 4.3.0-4.3.2 - breadcrumb not working correctly with nav_hide pages
Since Version 4.3.0 of VHS there is a problem with the viewhelper for <v:page.breadCrumb>.

It is not possible to use the breadcrumb menu correctly so that the hidden pages are not displayed in the menu. ( Typo3 BE Page Property <input value="1" data-formengine-input-name="data[pages][5][nav_hide]" onclick="document.editform['data[pages][5][nav_hide]'].value=this.checked?(document.editform['data[pages][5][nav_hide]'].value|1):(document.editform['data[pages][5][nav_hide]'].value&amp;0);TBE_EDITOR.fieldChanged('pages','5','nav_hide','data[pages][5][nav_hide]');" checked="checked" 1="" id="data_pages_5_nav_hide_0" type="checkbox"> )

Pages with page property hidden in menu will still be displayed. The option <v:page.breadCrumb showHiddenInMenu="0"> has no effect. It seems that the page property nav_hide is not processed. All pages are displayed, whether hidden in the menu or not.

The last correctly working Version is 4.2.0

### PR DESCRIPTION
**Pull request by xf-**
Resolves #1408

### CODE REVIEW

**Comment by cedricziel** [comment]
Looks good. Thx, @xf- !

File: ext_localconf.php [hunk 1]
```diff
@@ -82,7 +82,10 @@ function mb_scrub($s, $enc = null) { $enc = null === $enc ? mb_internal_encoding
...
 $GLOBALS['TYPO3_CONF_VARS']['SYS']['fluid']['namespaces']['v'] = ['FluidTYPO3\\Vhs\\ViewHelpers'];
 
-// add navigtion hide, url and urltype to fix the rendering of external url doktypes
+// add navigtion hide to fix menu viewHelpers (e.g. breadcrumb)
+$GLOBALS['TYPO3_CONF_VARS']['FE']['addRootLineFields'] .= (TRUE === empty($GLOBALS['TYPO3_CONF_VARS']['FE']['addRootLineFields']) ? '' : ',') . 'nav_hide';
+
+// add and urltype to fix the rendering of external url doktypes
 if (isset($GLOBALS['TCA']['pages']['columns']['urltype'])) {
-    $GLOBALS['TYPO3_CONF_VARS']['FE']['addRootLineFields'] .= (TRUE === empty($GLOBALS['TYPO3_CONF_VARS']['FE']['addRootLineFields']) ? '' : ',') . 'nav_hide,url,urltype';
+    $GLOBALS['TYPO3_CONF_VARS']['FE']['addRootLineFields'] .= ',url,urltype';
```
Comment (cedricziel) [inline ext_localconf.php]: Why do we allow a prepended comma here, but not above?

Comment (xf-) [reply ext_localconf.php]: above has a condition- if not empty use a `,`
Line 86 make sure a entry exists and a `,`  is needed = no condition.

**Comment by xf-** [comment]
fixed a typo

File: ext_localconf.php (hunk 1 above)
Comment (cedricziel) [reply ext_localconf.php]: Agreed.

**Pull request merged**